### ISSUE TITLE
Accessibility issue during "Import file" process.  

### ISSUE THREAD

**Issue by YaroRoodzit** [fgpv-vpgf/fgpv-vpgf#3530]
Tested build: v3.0.0-b5
The user cannot change the color of the layer when using only keyboard navigation. The color palette has to be selected  with the mouse.

**Comment by RyanCoulsonCA** [comment]
This seems to be a problem with the colour picker plugin that we're using, jQuery MiniColors. Looking at the documentation [here](https://labs.abeautifulsite.net/jquery-minicolors/#api), there doesn't seem to be any way to make the colour picker accessible. None of the examples on that page are accessible either. These are two ideas that could help us solve this problem:
1. We find a different colour picker plugin that is accessible, or
2. We find a way to make this colour picker accessible

Open to any other ideas or any methods of implementing one of the above two ideas.

**Comment by AleksueiR** [comment]
When this component was built, we couldn't find any good and accessible colour picker libraries. It was decided that generating a random colour and allowing users to enter colour codes through a text field would be sufficient.

@RyanCoulsonCA if you could find an accessible library to replace MiniColors, that would be the best approach.

**Comment by RyanCoulsonCA** [comment]
So after a few days of searching, it looks like there still aren't really any good, accessible colour picker libraries out there. I agree that our current method of allowing users to enter colour codes into a text box is sufficient in terms of accessibility, so let's keep it the way it is. 🎉

**Comment by AleksueiR** [comment]
What about this one? 
http://colourpicker.jasonhibbs.co.uk

It has keyboard support and there aria attributes on the element (always a good sign). It's a bit old, but it seems to work.

**Comment by RyanCoulsonCA** [comment]
Yup you're right, I looked at that one yesterday but it wasn't working when I pressed the arrow keys. Today it works fine, so I was probably doing something wrong. If no one has any objections, I can try and switch over to this one.

**Comment by RyanCoulsonCA** [comment]
It doesn't look like this colour picker is on npm, so if we were to use it I would need to add all of the JavaScript files to the viewer repo. Is this something that we're ok with doing? If so, where would be a good place to include these files?

**Comment by james-rae** [comment]
Another potential thing we can do is fork it's [github repo](https://github.com/jasonhibbs/colourpicker) and we point to that in our package file.  Forking leaves a trail back to the original creator, but protects us in case they delete their repo. We can also adjust the code and not require the author to PR it.

E.g. we did it [here](https://github.com/fgpv-vpgf/shapefile-js) and then point to it [here](https://github.com/fgpv-vpgf/geoApi/blob/v3.1.0-6/package.json#L23)

**Comment by AleksueiR** [comment]
Yes, forking under fgpv organization is the way to go.

**Comment by mpath** [comment]
Tested in Build: V3.2.0-b1
Issue resolved.
